modules/ROOT/pages/apikit-odatav4-extension-reference.adoc

```diff
@@ -90,124 +124,153 @@ Default configuration
-| Page Size a| Number |  |  | 
-| Config Ref a| ConfigurationProvider |  The name of the configuration to be used to execute this component |  | *x*
-| Streaming Strategy a| * <<RepeatableInMemoryStream>>
-* <<RepeatableFileStoreStream>>
-* non-repeatable-stream |  Configure if repeatable streams should be used and their behaviour |  | 
-| Entity Type Name a| String |  Entity type name specified by the Entity Data Model. |  | *x*
-| Entity Type Fields a| Array of String |  List of property fields from the Entity type. |  | *x*
-| Entity Type Keys a| Object |  Map of entity key names and values. |  | *x*
-| System Query Options a| Object |  System Query Options map. The grammar and syntax rules for system query options are defined in [OData-ABNF], in order to simplify usability we remove initial dollar sign to avoid escaping withing DataWeave, in example to access System Query Option $skip user should use the following expression #[attributes.systemQueryOptions.'skip'] |  | *x*
-| Search Transformation Enabled a| Boolean |  Enables $search transformation to WHERE clause. |  true | 
-| Strict Search Term Match a| Boolean |  Enables full term match for $search. If enabled, terms in WHERE clause are not enclosed between '%'. |  false | 
-| Target Variable a| String |  The name of a variable on which the operation output is placed. |  | 
-| Target Value a| String |  An expression that is evaluated against the operation output and the outcome of that expression is stored in the target variable. |  #[payload] | 
+| Configuration | String | Name of the configuration to use | | *x*
+| Entity Set Name | String | Table name in the database for the requested entity set |  | *x*
+| Database Schema | String | Database schema name to which the entity set belongs. If not empty, it is prefixed to the table name in the WHERE clause |  | 
+| Page Size | Number |  |  | 
+| Config Ref | ConfigurationProvider | Name of the configuration used to execute this component |  | *x*
+| Streaming Strategy | 
+
+* xref:apikit-odatav4-extension-reference#repeatable-in-memory-stream[RepeatableInMemoryStream]
+* xref:apikit-odatav4-extension-reference#repeatable-file-store-stream[RepeatableFileStoreStream]
+* non-repeatable-stream 
+
+| Configure if repeatable streams and their behavior are used |  | 
+| Entity Type Name | String | Entity Data Model specifies the entity type name |  | *x*
+| Entity Type Fields | Array of String | List of property fields from the Entity type |  | *x*
+| Entity Type Keys | Object | Map of entity key names and values |  | *x*
+| System Query Options | Object | System Query Options map. The grammar and syntax rules for system query options are defined in [OData-ABNF], to simplify usability the initial dollar sign is removed to avoid escaping withing DataWeave, in example to access System Query Option $skip user uses the following expression #[attributes.systemQueryOptions.'skip'] |  | *x*
```

Comment:
```suggestion
| System Query Options | Object | System Query Options map. The grammar and syntax rules for system query options are defined in [OData-ABNF]. To simplify usability, the initial dollar sign is removed to avoid escaping within DataWeave. For example, to access System Query Option, $skip user uses the following expression #[attributes.systemQueryOptions.'skip'] |  | *x*
```
Not sure if the suggestion is correct, the sentence is long and the last part is confusing. Please check.

Reply:
Great catch! Thanks!